standalone-metastore/metastore-server/pom.xml

```diff
@@ -474,23 +474,6 @@
               </target>
             </configuration>
           </execution>
-          <execution>
-            <id>generate-version-annotation</id>
-            <phase>generate-sources</phase>
-            <configuration>
-              <target>
-                <exec executable="bash" failonerror="true">
-                  <arg value="${basedir}/src/main/resources/saveVersion.sh"/>
-                  <arg value="${project.version}"/>
-                  <arg value="${hive.version.shortname}"/>
-                  <arg value="${basedir}/src"/>
-                </exec>
-              </target>
-            </configuration>
-            <goals>
-              <goal>run</goal>
-            </goals>
-          </execution>
```

Comment:
Do we know why was this introduced in the first place and if it is safe to remove?
If I understood well this is the main point of the fix, can you confirm?

Reply:
I think HIVE-20188 made the mistake to duplicate the code instead of moving it

Reply:
Also removed the script too, as it was duplicated as well

Reply:
Moved the `MetastoreVersionInfo` as suggested.
We do not use it without the `metastore-common`, so it was not causing any issues, but it is more logical this way